Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Views/SplashView.cs

```diff
@@ -0,0 +1,19 @@
+using Android.App;
+using Android.Content.PM;
+using MvvmCross.Core.ViewModels;
+using MvvmCross.Droid.Views;
+using MvvmCross.Platform;
+
+namespace MvvmSeed.Android.Views
+{
+    [Activity(MainLauncher = true, NoHistory = true, ScreenOrientation = ScreenOrientation.Portrait, Theme = "@style/Theme.Splash")]
```

Comment:
Besides jokes, I saw the behavior you mention @mvazquez-ms. Maybe is fine for the consumers but it will be great to have something that works for both scenarios. No big deal anyway.